iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java

```diff
@@ -84,8 +84,7 @@ public List<TableIdentifier> listTables(Namespace namespace) {
     }
     if (namespace.isEmpty()) {
```

Comment:
I actually wonder if this syntax is necessary at all. what we were doing here is when db is not given just list everything ... but isn't that really an anti-pattern? cc @HotSushi

Reply:
Added TODO.